src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -320,6 +314,15 @@ func (r logglyErrorReporter) Report(err error, fullMessage string, ctx map[strin
 	}
 }
 
+func shouldReport(location string) bool {
```

Comment:
This is the heart of the change.  Previously, we tried to deduplicate on message content, which isn't very reliable.  Now, we deduplicate on the location, for example 'flashlight.config: config.go:71'.

Also, rather than limiting by number of duplicates, we do it based on time now.